.github/workflows/vip_dynamic_rebuild.yaml

```diff
@@ -0,0 +1,108 @@
+name: dynamic_vip_rebuild
```

Comment:
separate dynamic vip build because this one is workflow dispatch, so we don't want it to be a matrix with vip_rebuild.